src/main/java/movie/hibernate/controller/impl/UserControllerImp.java

```
@@ -9,14 +9,18 @@
 
 public class UserControllerImp implements UserController {
 
-  private UserService userService ;
+  private UserService userService;
 
   public UserControllerImp(UserService userService) {
     this.userService = userService;
   }
 
   public User getUser(String login) {
-    return userService.getUserByLogin(login);
+    try {
+      return userService.getUserByLogin(login);
+    } catch (RuntimeException e) {
```

Empty catch block is anti-pattern "Error hiding: Catching an error message before it can be shown to the user and either showing nothing or showing a meaningless message"
See: https://en.wikibooks.org/wiki/Introduction_to_Software_Engineering/Architecture/Anti-Patterns